lib/va_profile/configuration.rb

```diff
@@ -12,7 +12,9 @@ def self.base_request_headers
     end
 
     def connection
-      @conn ||= Faraday.new(base_path, headers: base_request_headers, request: request_options) do |faraday|
+      ssl_enabled = Rails.env.production?
+      @conn ||= Faraday.new(base_path, headers: base_request_headers, request: request_options,
+                                       ssl: { verify: ssl_enabled }) do |faraday|
         faraday.use      :breakers
```

Comment:
This will allow devs to easily connect to VAProfile locally. At the moment, I have had to add` ssl: { verify: false}` before I can boot the vets-api servers locally. Rails.env == production in dev, staging, sandbox and prod.